### PR TITLE
fix: editLocalized and replyLocalized docs for i18n

### PR DESCRIPTION
**Pull request by IgnacioNMiranda**
## Fix
* Correct the usage instructions for `editLocalized` and `replyLocalized` in i18n getting started page.

### CODE REVIEW

**Comment by IgnacioNMiranda** [comment]
@all-contributors please add @IgnacioNMiranda for bug

**Review by favna — APPROVED** [review]

File: docs/Guide/plugins/i18next/getting-started.mdx [hunk 1]
```diff
@@ -200,7 +200,7 @@ export class PingCommand extends Command {
   }
 
   public async messageRun(message: Message) {
-    await replyLocalized(message, 'commands/ping:success_args', { latency: ws.ping });
+    await replyLocalized(message, { keys: 'commands/ping:success_with_args', formatOptions: { latency: 5 } });
```
Comment (favna) [inline docs/Guide/plugins/i18next/getting-started.mdx:203]:
```suggestion
    await replyLocalized(message, { keys: 'commands/ping:success_with_args', formatOptions: { latency: ws.ping } });
```

File: docs/Guide/plugins/i18next/getting-started.mdx [hunk 2]
```diff
@@ -180,7 +180,7 @@ export class PingCommand extends Command {
   }
 
   public async messageRun(message: Message) {
-    await editLocalized(message, 'commands/ping:success_args', { latency: ws.ping });
+    await editLocalized(message, { keys: 'commands/ping:success_with_args', formatOptions: { latency: 5 } });
```
Comment (favna) [inline docs/Guide/plugins/i18next/getting-started.mdx:183]:
```suggestion
    await editLocalized(message, { keys: 'commands/ping:success_with_args', formatOptions: { latency: ws.ping } });
```

**Review by favna — CHANGES_REQUESTED** [review]

File: docs/Guide/plugins/i18next/getting-started.mdx (hunk 2 above)
Comment (kaname-png) [reply docs/Guide/plugins/i18next/getting-started.mdx]: Where does the variable ws come from?

**Pull request merged**